pkg/logs/internal/tailers/journald/docker.go

```diff
@@ -43,7 +45,7 @@ func (t *Tailer) getContainerTags(containerID string) []string {
 
 // initializeTagger initializes the tag collector.
 func (t *Tailer) initializeTagger() {
-	err := tagger.Init()
+	err := tagger.Init(context.TODO())
```

Comment:
should we update `initializeTagger()` to provide a context?

Reply:
Ok I checked https://github.com/DataDog/datadog-agent/blob/55da4f3c2a2dfce0f7e57c1fcb5c590636c53a88/pkg/logs/internal/tailers/journald/tailer.go#L76 
and it seems to be a big change to make it properly.